test/extensions/health_checkers/redis/config_test.cc

```diff
@@ -39,7 +39,7 @@ TEST(HealthCheckerFactoryTest, DEPRECATED_FEATURE_TEST(CreateRedisDeprecated)) {
   EXPECT_NE(
       nullptr,
       dynamic_cast<CustomRedisHealthChecker*>(
-          factory.createCustomHealthChecker(Upstream::parseHealthCheckFromV2Yaml(yaml), context)
+          factory.createCustomHealthChecker(Upstream::parseHealthCheckFromV3Yaml(yaml), context)
```

Comment:
FYI, I don't think we want to change the parsing inside tests marked `DEPRECATED_FEATURE_TEST`. These should be v2 configs.

Reply:
Ok. I will update it.